src/tests/ftest/launch.py

```diff
@@ -192,6 +192,37 @@ def set_test_environment(args):
     os.environ["CRT_CTX_SHARE_ADDR"] = "0"
     os.environ["OFI_INTERFACE"] = os.environ.get("OFI_INTERFACE", interface)
 
+    # Update the default provider,domain based on the OFI_INTEFACE
+    print("Before Environment:")
+    for name in ("OFI_INTERFACE", "OFI_DOMAIN", "CRT_PHY_ADDR_STR"):
+        print(" {0:>20}: {1}".format(name, os.environ.get(name, "No env")))
+    fi_info_path = os.path.join(os.path.sep, bin_dir, "fi_info")
+
+    print("fi_info : ")
+    out = get_output(fi_info_path)
```

Comment:
Remember, `get_output()` wants a list, not a string.

```suggestion
    out = get_output([fi_info_path])
```

Reply:
Done.